opennms-base-assembly/src/main/filtered/etc/trend-configuration.xml

```diff
@@ -0,0 +1,232 @@
+<trend-configuration>
+
+    <trend-definition name="nodes">
+        <title>Nodes</title>
+        <subtitle>w/ Outages</subtitle>
+        <visible>true</visible>
+        <icon>glyphicon-fire</icon>
+        <trend-attributes>
+            <trend-attribute key="sparkWidth" value="100%"/>
+            <trend-attribute key="sparkHeight" value="35"/>
+            <trend-attribute key="sparkChartRangeMin" value="0"/>
+            <trend-attribute key="sparkLineColor" value="white"/>
+            <trend-attribute key="sparkLineWidth" value="1.5"/>
+            <trend-attribute key="sparkFillColor" value="#88BB55"/>
+            <trend-attribute key="sparkSpotColor" value="white"/>
+            <trend-attribute key="sparkMinSpotColor" value="white"/>
+            <trend-attribute key="sparkMaxSpotColor" value="white"/>
+            <trend-attribute key="sparkSpotRadius" value="3"/>
+            <trend-attribute key="sparkHighlightSpotColor" value="white"/>
+            <trend-attribute key="sparkHighlightLineColor" value="white"/>
+        </trend-attributes>
+        <descriptionLink>outage/list.htm?outtype=current</descriptionLink>
+        <description>${intValue[23]} NODES WITH OUTAGE(S)</description>
+        <query>
+            <![CDATA[
```

Comment:
Can we add integration tests to verify the default queries in the configuration file?

Otherwise, these may be forgotten if/when we make a schema change.